File: app/src/main/java/com/dante/diary/net/GsonConfig.java
package com.dante.diary.net;

import com.google.gson.Gson;
import com.google.gson.GsonBuilder;

/**
 * Created by yons on 17/3/6.
 */

public class GsonConfig {

    /**
     * created : 2010-03-19 10:32:26
     */
    public static final Gson gson = new GsonBuilder()
            .setDateFormat("yyyy-MM-dd HH:mm:ss")
            .create();

}

File: app/src/main/java/com/example/myapplication/net/GsonConfig.kt
package com.example.myapplication.net

import com.google.gson.Gson
import com.google.gson.GsonBuilder

object GsonConfig {

    val dateFormat = "yyyy-MM-dd HH:mm:ss"

    val gson: Gson = GsonBuilder()
        .setDateFormat(dateFormat)
        .create()


}